api/exp/livesessions.go

```diff
@@ -98,6 +126,24 @@ func (s *AntidoteAPI) ListLiveSessions(ctx context.Context, _ *empty.Empty) (*pb
 	return &pb.LiveSessions{Items: lsAPIs}, nil
 }
 
+// UpdateLiveSessionPersistence updates the persistence flag in the session database
+func (s *AntidoteAPI) UpdateLiveSessionPersistence(ctx context.Context, persistence *pb.SessionPersistence) (*empty.Empty, error) {
+	span := ot.StartSpan("api_livesession_persist", ext.SpanKindRPCClient)
+	defer span.Finish()
+
+	_, err := s.Db.GetLiveSession(span.Context(), persistence.SessionID)
+	if err != nil {
+		return &empty.Empty{}, errors.New("livesession not found")
+	}
+
+	err = s.Db.UpdateLiveSessionPersistence(span.Context(), persistence.SessionID, persistence.Persistent)
+	if err != nil {
+		return &empty.Empty{}, errors.New("Unable to update persistence value in record")
```

Comment:
Given that this is more of a "superuser" function, let's just return the `err` that we get from the underlying db function, so the client can see what's going on

Reply:
done